pkgs/by-name/hi/hidviz/package.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, stdenv
+# Package dependencies
+, qt6
+, libusb1
+, protobuf
+, asio
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hidviz";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "hidviz";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-9crHFYVNNxJjwJojwqB8qdAGyr1Ieux9qC3m3rpIJw0=";
+  };
+
+  dontWrapQtApps = true;
+
+  preConfigure = ''
+    substituteInPlace libhidx/cmake_modules/Findasio.cmake --replace-fail '/usr/include/asio' '${lib.getDev asio}/include/asio'
+  '';
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    qt6.qttools
+  ];
+  buildInputs = [
+    qt6.qtwebengine
+    libusb1
+    protobuf
+    asio
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/hidviz/hidviz";
+    description = "A GUI application for in-depth analysis of USB HID class devices.";
+    license = licenses.gpl3;
```

`licenses.gpl3` is discouraged due to ambiguity, and `licenses.gpl3Only` (i.e. later GPL versions are not used) or `licenses.gpl3Plus` ("version 3 of the License, or (at your option) any later version") are encouraged instead

Looking at the upstream code, this seems to be `gpl3Plus`

```suggestion
    license = licenses.gpl3Plus;
```